frontend/src/components/navigation/SideBar.vue

```diff
@@ -28,12 +29,24 @@ export default {
   name: 'SideBar',
   data () {
     return {
-      drawer: false
+      drawer: false,
+      dirty: false,
+      minimized: false
     }
   },
   computed: {
-    mini () {
-      return !this.$vuetify.breakpoint.mdAndUp
+    mini: {
+      get: function () {
+        if (this.dirty) {
+          return this.minimized
+        } else {
+          return !this.$vuetify.breakpoint.mdAndUp
+        }
+      },
+      set: function (value) {
+        this.dirty = true
+        this.minimized = value
+      }
```

Comment:
Diese Logik mit `dirty` und setter verhält sich soweit ich sehe wieder genau gleich wie wenn `mini` einfach ein Eintrag in `data` mit default wäre. Gibt es einen Unterschied den ich übersehe oder könnte man das wieder vereinfachen?

Reply:
Sobald du die Sidebar ein oder ausklappst, reagiert sie nicht mehr auf die Breakpoints. (Da man offensichtlich die Sidebar ein oder ausblenden will.

Reply:
Ach so, das stimmt. In dem Fall würde ich es so vorschlagen:
```js
data: {
  overrideExpanded: null
},
computed: {
  mini () {
    return this.overrideExpanded !== null ? !this.overrideExpanded : !this.$vuetify.breakpoint.mdAndUp
  }
} 
```
Und im `<template>` überall nur in `overrideExpanded` schreiben. (overrideExpanded statt overrideMini weil du dann immer noch leicht `overrideExpanded = !overrideExpanded`  schreiben kannst)